### PR TITLE
Fix and bugs - dev

### PR DESCRIPTION
**Pull request by Mati0x**
#375

### CODE REVIEW

File: apps/web/components/DisputeButton.tsx [hunk 1]
```diff
@@ -231,8 +232,8 @@ export const DisputeButton: FC<Props> = ({ proposalData }) => {
   };
 
   const content = (
-    <div className="flex flex-col gap-10 lg:min-w-[800px]">
-      {isDisputed || proposalStatus === "rejected" ?
```
Comment (Corantin) [inline apps/web/components/DisputeButton.tsx:235]: I'd like to see why the proposal is rejected, so we need to display the dispute button for this

File: apps/web/components/DisputeButton.tsx [hunk 2]
```diff
@@ -468,7 +464,7 @@ const DisputeMessage = ({
         )}
       </div>
       <div className="chat-bubble shadow-lg bg-neutral-200">
-        {dispute.metadata?.reason ?? disputeMetadata?.reason}
```
Comment (Corantin) [inline apps/web/components/DisputeButton.tsx:471]: Metadata can be undefined until the subgraph index the ipfs data source so we need to fallback to the disputeMetadata reason in this case

File: apps/web/components/DisputeButton.tsx [hunk 3]
```diff
@@ -410,7 +406,7 @@ const DisputeMessage = ({
     ProposalDispute,
     "id" | "challenger" | "context" | "createdAt"
   > & {
-    metadata?: Pick<ProposalDisputeMetadata, "reason"> | null;
```
Comment (Corantin) [inline apps/web/components/DisputeButton.tsx:413]: Need to be reverted, see the next comment

File: apps/web/components/Statistic.tsx [hunk 4]
```diff
@@ -17,19 +17,21 @@ export const Statistic = ({
...
-        className={"tooltip flex cursor-default items-center max-w-sm"}
-        data-tip={tooltip}
-      >
-        {icon && <div className="mr-1">{icon}</div>}
-        {label && (
-          <p className="first-letter:uppercase">
-            {label}: {count}
-          </p>
-        )}
+    <>
```
Comment (Corantin) [inline apps/web/components/Statistic.tsx:20]: no need fragments

File: apps/web/components/DisputeButton.tsx (hunk 2 above)
Comment (Mati0x) [reply apps/web/components/DisputeButton.tsx:471]: need me to do something here Goss ??

**Review by Corantin — APPROVED** [review]

**Review by Corantin — APPROVED** [review]

**Pull request merged**